moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp

```diff
@@ -57,6 +58,8 @@ class RobotInterfacePython : protected py_bindings_tools::ROScppInitializer
     py_bindings_tools::ROScppInitializer()
   {
     robot_model_ = planning_interface::getSharedRobotModel(robot_description);
+    if (!robot_model_)
+      throw std::runtime_error("RobotInterfacePython: invalid robot model");
```

Comment:
For some reason I can't find the definition of `planning_interface::getSharedRobotModel`, but does it not return more meaningful info when the model file is "invalid"? I'm afraid how useful for the developers to debug to be told that the model is invalid. If it only returns null obj when robot model is invalid, then this error handling message is perfectly fine (and we probably want to improve getSharedRobotModel to throw exception with more info).